endpoints/getting-started-grpc/helloworld/helloworld.proto

```diff
@@ -0,0 +1,52 @@
+// Copyright 2015, Google Inc.
```

Comment:
These are copied verbatim from the gRPC samples, so keeping their license headers intact.